//package com.sincetimes.redwar.game.comnunicate.protocol;
message RseObtainHangarsHelp
{
	repeated HangarsHelpList HangarsHelpList = 1;
}

message HangarsHelpList
{
	optional int32 ret = 1;
}